docs(common): contributing guide

eunjae-lee opened the pull request:
## What?

This PR updates the CONTRIBUTING.md 

[View the rendered file →](https://github.com/storyblok/field-plugin/blob/EXT-1256-contributing-readme/CONTRIBUTING.md)

## Why?

JIRA: EXT-1256

eunjae-lee commented:
Hi @BibiSebi and @demetriusfeijoo 
This is a new CONTRIBUTING.md guide for those who want to contribute to this SDK. Feel free to leave lots of questions, comments, and suggestions here 🙌🏻

BibiSebi commented on CONTRIBUTING.md:
```diff
@@ -1,19 +1,155 @@
+
+Once a release is created, one of the two GitHub Actions - [.github/workflows/npm-publish-library.yml](https://github.com/storyblok/field-plugin/blob/main/.github/workflows/npm-publish-library.yml) or [.github/workflows/npm-publish-cli.yml](https://github.com/storyblok/field-plugin/blob/main/.github/workflows/npm-publish-cli.yml) - will run and deploy it to NPM.
+
+### Order of releases
```
Should we also add a section explaining what is important for the PR. Like explaining what has been changes and some guidelines to not submit a huge PR or something similar

BibiSebi commented on CONTRIBUTING.md:
```diff
@@ -1,19 +1,155 @@
+
+## Release
+
+You can release either `@storyblok/field-plugin` or `@storyblok/field-plugin-cli`. To begin the release process, run the following command on the `main` branch:
```
should contributors have the right to release?

eunjae-lee commented:
@BibiSebi thanks for the review! I've addressed your feedback :) 
https://github.com/storyblok/field-plugin/pull/225/commits/9f42d8d0a6017dc6bda207733c8b68da1b1abd05

BibiSebi reviewed (approved):
Awesome job @eunjae-lee 👏 let's merge it and see if there is any feedback once someone starts contributing
